get_gerrit_last_comment_by_id.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+command -v jq >/dev/null 2>&1 || { echo >&2 "I require jq but it's not installed. Aborting."; exit 1; }
+
+usage() { 
+    echo "Usage: $0 [-u|--gerrit_server] [-p|--port] [-i|--id] [-s|--ssh_key_path]"; 
```

Comment:
it is `--port NUM` right, not jus `--port`?

Reply:
You're right. Change done.